cypress/e2e/court-cases/notes/add.cy.ts

```diff
@@ -2,8 +2,9 @@ import type { TestTrigger } from "../../../../test/utils/manageTriggers"
 import User from "services/entities/User"
 import a11yConfig from "../../../support/a11yConfig"
 import logAccessibilityViolations from "../../../support/logAccessibilityViolations"
+import hashedPassword from "../../../fixtures/hashedPassword"
 
-describe("Home", () => {
+describe("Case details", () => {
   context("720p resolution", async () => {
     const users: Partial<User>[] = await Promise.all(
       Array.from(Array(5)).map(async (_value, idx) => {
```

Comment:
Not relevant to this PR. Something to fix later.
Using `Promise.all()` and async/await seem redundant here.

Reply:
I'll make a note of this! 👍

Reply:
Resolved it [1372962](https://github.com/ministryofjustice/bichard7-next-ui/pull/101/commits/13729629fb88a4e2b993c807487e85856774d750)